zproject/urls.py

```diff
@@ -220,6 +218,12 @@
         {'PATCH': 'patch_bot_backend',
          'DELETE': 'deactivate_bot_backend'}),
 
+) + patterns('zerver.views.upload',
+    url(r'^user_uploads/(?P<realm_id_str>(\d*|unk))/(?P<filename>.*)', 'rest_dispatch',
+            {'GET': 'api_get_uploaded_file'}),
```

Comment:
Do you want this to be a like a general case so that other endpoints can be added later?

Also if we are having only one endpoint for getting the files then we can remove the need having multiple endpoints for uploading files too.